web/services/AI/ai_model.py

```
@@ -0,0 +1,180 @@
+        else:
+            raise ValueError("No JSON object found in response")
+        
+        # print("Parsed response:", response_object)
+        return response_object
+    
+    except Exception as e:
+        print(f"Error in AI processing: {e}")
+        return {
+            "degree": 0,
+            "student_feed_back": "Error: Could not evaluate the answer.",
+            "teacher_feed_back": "Error: AI failed to process the response."
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)